nvflare/fuel/utils/component_classes.json

```diff
@@ -475,9 +520,6 @@
     "TFModelPersistor": [
         "nvflare.app_opt.tf.model_persistor"
     ],
-    "MLflowReceiver": [
-        "nvflare.app_opt.tracking.mlflow.mlflow_receiver"
-    ],
```

Comment:
not sure why this is gone though

Reply:
Added back. It's due to a class loading exception.